source/MaterialXTest/MaterialXGenShader/GenShaderUtil.cpp

```diff
@@ -324,8 +328,23 @@ void shaderGenPerformanceTest(mx::GenContext& context)
     context.registerSourceCodeSearchPath(libSearchPath);
 
     // Enable Color Management
-    mx::ColorManagementSystemPtr colorManagementSystem =
+    mx::ColorManagementSystemPtr colorManagementSystem;
+#ifdef MATERIALX_BUILD_OCIO
+    try
+    {
+        auto config = OCIO::Config::CreateFromBuiltinConfig("ocio://studio-config-latest");
```

Comment:
I'm not an OCIO expert - but does this pull some pre-baked config from inside OCIO.  Will people need to provide custom OCIO configs at some point?

Reply:
@ld-kerley , the PR supports use of arbitrary OCIO configs, but to keep things simple for the purposes of this unit test, Jerry is using one of the configs that's built into the OCIO library itself.

In the real world, apps could create the config using OCIO::Config::CreateFromFile(path) rather than CreateFromBuiltinConfig.